.github/workflows/moodle-plugin-ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   # push only for branches (ignore tags)
   push:
     branches:
-      - "main"
+      - "**"
```

Comment:
I undid this since I thought it makes not sense to run the pipeline on every push on every branch. Instead the workflow is: pushing outside main does nothing, once pr is opened the pipeline will run. When merging to main (push) the pipeline is being run. What do you think of that?

Reply:
FYI: https://github.com/bschur/moodle-qtype_uml/issues/52

Reply:
The problem is that contributors who fork the repository and then create branches to create pull requests (which probably aren't called 'main'), those branches do not get any automated checks. That's a pity.

Reply:
Thanks for the declaration. That's an good argument too. I'll approve it like this 👍